integration-tests/tests/utils/utils.py

```diff
@@ -0,0 +1,25 @@
+import os
+import subprocess
+from pathlib import Path
+from typing import List
+
+
+def diff_equal(path1: Path, path2: Path) -> None:
```

Comment:
for name, how about diff_path?

Reply:
as discussed with Kirk we are going to name it `is_dir_tree_content_equal`